opm/porsol/mimetic/IncompFlowSolverHybrid.hpp

```diff
@@ -155,7 +155,7 @@ namespace Opm {
         ///    and usually @code T @endcode is an alias for @code
         ///    double @endcode.
         template<typename T>
-        class axpby : public std::binary_function<T,T,T> {
+        class axpby {
```

Comment:
Cool.  Would you mind making the same change to [`MaxAbs`](https://github.com/OPM/opm-upscaling/blob/9219235c5b952452f213068c978203273b322570/opm/core/transport/implicit/NormSupport.hpp#L47), [`SumAbs`](https://github.com/OPM/opm-upscaling/blob/9219235c5b952452f213068c978203273b322570/opm/core/transport/implicit/NormSupport.hpp#L59), and [`Euclid`](https://github.com/OPM/opm-upscaling/blob/9219235c5b952452f213068c978203273b322570/opm/core/transport/implicit/NormSupport.hpp#L71) too?  That way we can close Issue #374 as well.

Reply:
done.